modules/persistent-storage-csi-efs-cross-account.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-efs-cross-account_{context}"]
+= AWS EFS CSI cross account support
+
+Cross account support allows you to have an {product-title} cluster in one AWS account and mount your file system in another AWS account using the AWS Elastic File System (EFS) Container Storage Interface (CSI) driver.
+
+[NOTE]
+====
+Both the {product-title} cluster and EFS file system should be in the same region.
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster with administrator rights
+
+* Two valid AWS accounts
+
+.Procedure
+
+The following procedure demonstrates how to set up:
+
+* {product-title} cluster in AWS account A
+
+* Mount an AWS EFS file system in account B
+
+To use AWS EFS across accounts:
+
+. Install {product-title} cluster with AWS account A and install the EFS CSI Driver Operator.
+
+. Create an EFS volume in AWS account B:
+
+.. Create a test virtual private cloud (VPC) called “test-efs-vpc” with CIDR “172.20.0.0/16” and subnet for the AWS EFS volume.
```

Comment:
i would avoid using the term "test" in our naming convention.
Also it's up to the customer to choose his CIDR, 172.20.0.0/16 is just and example. The wording suggests it has to be set to this value. thoughts?

Reply:
I'm pretty sure you don't need that value. I've indicated that it is an example.